pkg/apis/operator/v1alpha1/tektonresult_defaults.go

```diff
@@ -21,4 +21,8 @@ import (
 )
 
 func (tp *TektonResult) SetDefaults(ctx context.Context) {
+	// Deprecate TLSHostnameOverride
+	if tp.Spec.TLSHostnameOverride != "" {
+		tp.Spec.TLSHostnameOverride = ""
+	}
```

Comment:
@khrm how do you handle this on existing `TektonResult` CR?

Reply:
We don't handle this for existing CR, do we? We are giving a notice that we have deprecated the field. So the user should remove the field but if this field comes, we remove it in this release.

Reply:
Isn't this the same pattern that we follow in other components? Do we do something different in the operator?

Reply:
this change is t avoid upgrade failure right ?

Reply:
Yes.

Reply:
even if on old resources, someone will edit it will remove from that